.travis.yml

```diff
@@ -23,6 +26,10 @@ matrix:
       env: COMPOSER_OPTIONS="--prefer-lowest --prefer-stable"
     - php: 7.0
       env: xdebug="yes" COMPOSER_OPTIONS=""
+      php: 7.1
+      env: xdebug="yes" COMPOSER_OPTIONS=""
+      php: 7.2
+      env: xdebug="yes" COMPOSER_OPTIONS=""
```

Comment:
I would not add these to builds. Just using 7.1 and 7.2 without Xdebug is enough.